app/assets/javascripts/app/utils/countdown-timer.js

```diff
@@ -11,7 +10,6 @@ export default (targetSelector, timeLeft = 0, interval = 1000) => {
     countdownTarget.innerHTML = msFormatter(remaining);
 
     if (remaining <= 0) {
```

Comment:
Ok so based on my exploration of all the different flows, and testing, etc, including the 'account locked' screen, I am satisfied with this behavior.

On the 'account locked' screen, it will still pop up a modal, where clicking "continue" takes the user back to the home screen (as opposed to automatically redirecting).